src/components/ProfileHeaderMenuItem/ProfileHeaderMenuItem.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { afterEach } from "vitest";
+
+import { ProfileHeaderMenuItem } from "./ProfileHeaderMenuItem";
+
+afterEach(() => {
```

Comment:
The global test setup file already has this `afterEach` cleanup block.  This one can be removed.